app/workers/form1010cg/submission_job.rb

```diff
@@ -1,21 +1,40 @@
 # frozen_string_literal: true
 
+require 'sidekiq/monitored_worker'
+
 module Form1010cg
   class SubmissionJob
+    STATSD_KEY_PREFIX = "#{Form1010cg::Auditor::STATSD_KEY_PREFIX}.async."
     include Sidekiq::Worker
+    include Sidekiq::MonitoredWorker
     include SentryLogging
 
     sidekiq_options(retry: 14)
 
+    sidekiq_retries_exhausted do |msg, _e|
+      StatsD.increment("#{STATSD_KEY_PREFIX}failed_no_retries_left", tags: ["claim_id:#{msg['args'][0]}"])
```

Comment:
StatsD.increment("#{STATSD_KEY_PREFIX}.failed_no_retries_left

Reply:
the constant `STATSD_KEY_PREFIX` includes a trailing dot

Reply:
@lihanli Thanks for clearing that up.